chime-notifier/README.md

```diff
@@ -0,0 +1,53 @@
+## AWS Health Chime Notifier
+
+### Description
+
+This tool can be used to post alerts to a Chime room when AWS Health events are generated by using AWS Lambda and Amazon CloudWatch Events. 
+
+### Chime Setup
+Follow these steps to configure the webhook in Chime:
+
+1. Click the cog wheel from your Chime room 
+
+2. Select "Manage WebHooks".
+
+3. Create Webhook
+
+4. Copy the webhook URL and use it as the value for the *HookURL* parameter in the Cloudformation template. 
+
+Security Note: WebHooks should be treated like passwords and should not be shared publicily. 
+
+### AWS setup using CloudFormation
+
+#### CloudFormation
+
+Choose **Launch Stack** to launch the template in the US East (N. Virginia) Region in your account.
+ 
+The CloudFormation template requires the following parameter:
+ 
+*HookURL* - Incoming web hook url from Slack setup.
+
+
+### AWS Manual Setup
+
+1. Create an IAM role for the Lambda function to use. Attach the [IAM policy](IAMPolicy) to the role in the IAM console.
+
+Documentation on how to create an IAM policy is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html
+Documentation on how to create an IAM role for Lambda is available here: http://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-service.html#roles-creatingrole-service-console
+
+2. Create a Lambda Python function by using the [sample](LambdaFunction.py) provided and choose the IAM role created in step 1. Add an enviroment variable with key CHIMEWEBHOOK and the webhook URL from the Chime setup above as value.
```

Comment:
When you create a Python Lambda function the handler defaults to 'lambda_function.lambda_handler'. In the sample function code provided the handler is called 'handler'. Either update the code to use 'lambda_handler' or add a step here to change the handler name.

Reply:
updated to lambda_handler

Reply:
updated to lambda_handler